content/en/apps/tutorials/form-properties.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "Setting Form Properties"
+linkTitle: Form Properties
+weight: 8
+description: >
+  Setting Form Properties
+relatedContent: >
+  apps/reference/forms/app/#formsappform_namepropertiesjson
+
+---
+
+The form properties file contains meta information related to the App form.
+
+## Purpose of the Tutorial
+
+This tutorial will take you through how to write the `<form_name>.properties.json` file.
+
+You will be adding meta-data and context to an assessment workflow that allows Community Health Workers to conduct a health assessment for children under the age of 5.
```

Comment:
Can you add a sentence on the value of a user adding meta-data and context?

Reply:
Updated.